libs/SalesforceSDKCore/SalesforceSDKCore/Classes/UserAccount/SFUserAccountManager.m

```diff
@@ -1544,14 +1544,15 @@ - (void)retrievedIdentityData:(SFSDKAuthSession *)authSession {
     NSAssert(authSession.identityCoordinator.idData != nil, @"Identity data should not be nil/empty at this point.");
     SFIdentityCoordinator *identityCoordinator = authSession.identityCoordinator;
     BOOL hasMobilePolicy = identityCoordinator.idData.mobilePoliciesConfigured;
+    int lockTimeout = identityCoordinator.idData.mobileAppScreenLockTimeout;
     
     __weak typeof(self) weakSelf = self;
     [self dismissAuthViewControllerIfPresentForScene:authSession.oauthRequest.scene completion:^{
           __strong typeof(weakSelf) strongSelf = weakSelf;
         [strongSelf finalizeAuthCompletion:authSession];
         if (authSession.authInfo.authType != SFOAuthTypeRefresh) {
-            [[SFScreenLockManager shared] storeMobilePolicyWithUserAccount:self.currentUser hasMobilePolicy:hasMobilePolicy];
-            [[SFScreenLockManager shared] handleAppForeground];
+            [[SFScreenLockManager shared] storeMobilePolicyWithUserAccount:self.currentUser hasMobilePolicy:hasMobilePolicy lockTimeout:lockTimeout];
+            [[SFScreenLockManager shared] handleAppForegroundWithLockTimeout:NO];
```

Comment:
I don't quite understand why the `lockTimeout` boolean is needed.  Just to force lock on initial login?

Reply:
yup, though since you mentioned that it should only be with a passcode user, this should be updated. Or could go with the other suggestion to  reset the timestamp to 0 in `storeMobilePolicy`